src/content-helper/editor-sidebar/excerpt-suggestions/component-panel.tsx

```diff
@@ -252,22 +275,25 @@ const PostExcerptSuggestions = () => {
 					help={ wordCountString ? wordCountString : null }
 				/>
 			</div>
-			<Button
-				href={ __(
-					'https://wordpress.org/documentation/article/page-post-settings-sidebar/#excerpt',
-					'wp-parsely',
-				) }
-				target="_blank"
-				variant="link"
-			>
-				{ __( 'Learn more about manual excerpts', 'wp-parsely' ) }
-				<Icon
-					icon={ external }
-					size={ 18 }
-					className="parsely-external-link-icon"
-				/>
-			</Button>
-			<div className="wp-parsely-excerpt-generator">
+
+			{ isDocumentSettingPanel && (
+				<Button
+					href={ __( // eslint-disable-line @wordpress/i18n-text-domain
+						'https://wordpress.org/documentation/article/page-post-settings-sidebar/#excerpt'
```

Comment:
Curious why we aren't using a text domain here. Can you explain?

Reply:
This string is part of the core WordPress translation, as this piece of code was extracted from the original excerpt textfield. So in order for the URL to be translated to the correct URL depending on the language, it needs to be in the core domain, ie, no domain.

Reply:
Thank you!